Monika-After-Story/MonikaModDev
# Pose 7

#4304 

wait for #4582 before doing review

# Key Changes
* Adds pose 7 - (down left + rest point right)

# Testing
* verify pose looks ok and is selectable in exp previewer
* verify you can create pose 7 stuff in the sprite creation tool
